### DISCUSSION TITLE
Invalid TimeField vaildation

### DISCUSSION THREAD

**Discussion by random1st** [encode/django-rest-framework#9540] [General] [answered]
## Checklist

An amendment was published in October 2022, featuring minor technical clarifications and attempts to remove ambiguities in definitions. The most significant change, however, was the reintroduction of the "24:00:00" format to refer to the instant at the end of a calendar day.

with 24:00:
`"Time has wrong format. Use one of these formats instead: hh:mm[:ss[.uuuuuu]].`

**Comment by lovelydinosaur** [comment]
> An amendment was published in October 2022

→(?)

**Reply by random1st** [reply]
It means ISO 8601 latest edition

**Reply by random1st** [reply]
https://github.com/python/cpython/issues/124257

**Reply by lovelydinosaur** [reply, marked as answer]
Support for this in cpython would be a prerequisite to us considering support for it here.

(Thanks, TIL)
